### DISCUSSION TITLE
Problème avec l'exécution du jar pour le projet

### DISCUSSION THREAD

**Discussion by GFragniere** [heig-vd-dai-course/heig-vd-dai-course#432] [Q&A] [answered]
Bonjour,
J'ai comme qui dirait un problème lors de l'exécution du .jar après compilation par maven dans le cadre du projet. Mon collègue n'a pas ce problème, mais quand j'essaie de lancer le programme, j'obtiens l'erreur suivante:
![image](https://github.com/user-attachments/assets/9b26ebc8-7928-47d1-81d9-2afd2540a59b)
et je ne suis pas sûr de comprendre son sens, ni d'où elle provient 😓 
Merci d'avance

**Comment by GFragniere** [comment]
Petite précision, le fichier sois disant non-existant existe bien

**Reply by ludelafo** [reply, marked as answer]
Bonjour @GFragniere,

Merci pour votre message.

Je n'ai jamais rencontré cette erreur jusqu'à maintenant donc je n'ai peut-être pas toutes les clés en main pour vous répondre. Vous pouvez néanmoins essayer les étapes suivantes :

- Désinstaller et réinstaller la version de Java à l'aide de SDKMAN! pour s'assurer que l'installation n'est pas corrompue
- Installer les paquets décrits dans le commentaire Stack Overflow suivant : <https://stackoverflow.com/questions/72049332/java-lang-unsatisfiedlinkerror-opt-jdk17-lib-libawt-xawt-so-libxrender-so-1#comment135499165_76282148> (j'ai cherché sur Google l'erreur "ubuntu java libxrender.so.1 not found")

Aussi, est-ce vous pouvez me remettre le lien de votre repo Git ?

Tenez-moi au courant si cela a résolu le souci !

**Reply by GFragniere** [reply]
Bonjour, 

J'ai essayé de faire une réinstallation de java sans succès, par contre l'exécution des différentes commandes à réglé le problème et me permet de pouvoir exécuter le problème.

Cependant, maintenant, IntelliJ semble avoir des difficultés pour faire de la résolution de nom, rien de très grave en somme.

Lien de notre repo Git: https://github.com/Killian-Viquerat/DAI_Practical_Work_1_Magic_Image_CLI

Bonne soirée, à demain!
